scripts/cassandra-integration-test.sh

```diff
@@ -2,29 +2,11 @@
 
 set -ex
 
-# Clean up before starting.
```

Comment:
what is the motivation for removing setup & cleanup? They are especially useful when running tests locally

Reply:
I thought to use GA's [service containers](https://docs.github.com/en/actions/guides/about-service-containers) for managing the containers (like we do in kafka integration test) so we can do parallel tests and don't need to run containers on two ports and duplicate things in integration test (as was done before). I feel this is lot cleaner approach but think otherwise, I can revert back to earlier way.